CHANGELOG.md

```diff
@@ -0,0 +1,35 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
```

Comment:
Do we want to use 1.0.0 or [1.1.0](https://keepachangelog.com/en/1.1.0/)?

Reply:
Lol I copied the text from their 1.1.0 site which references 1.0.0.

Reply:
lol, time for 1.1.1

Reply:
https://keepachangelog.com/en/1.1.1/ is a dead link 💯